src/templates/app/_layout/footer.jsx

```diff
@@ -140,6 +140,11 @@ const Footer = () => (
                                     { media: 'reddit',      href: 'https://www.reddit.com/user/Deriv_official/' },
                                 ]}
                             />
+                            <div className='trustlogo2'>
+                                <div className='trustpilot-widget' data-locale='en-US' data-template-id='5419b757fa0340045cd0c938' data-businessunit-id='58dc92220000ff00059f7dc4' data-style-height='25px' data-style-width='100%'>
+                                    <a href='https://www.trustpilot.com/review/binary.com' target='_blank' rel='noopener noreferrer'>Trustpilot</a>
```

Comment:
do we need to send `Trustpilot` to translations?

Reply:
it is third-party logo with dynamic text so think cannot translate, pls share your advice

Reply:
true